pom.xml

```diff
@@ -154,6 +155,16 @@
             <artifactId>spark-core_${scala.binary.version}</artifactId>
             <version>${spark.version}</version>
             <scope>provided</scope>
+            <exclusions>
```

Comment:
These exclusions are missing from `sbt`. Is that OK?

Reply:
Thanks for noticing. I have added the same excludes to sbt version as well.